.circleci/regenerate.py

```diff
@@ -45,6 +45,11 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
                     if (
                         (cu_version.startswith("rocm") and btype == "conda")
                         or (os_type == "linux" and btype == "wheel")
+                        or (
+                            os_type == "linux"
+                            and btype == "conda"
+                            and (python_version != "3.8" or cu_version != "cu116")
```

Comment:
IIUC we want to remove all linux conda jobs except the py3.8 cu116 build? I think this should be 
`and (python_version != "3.8" and cu_version != "cu116")` in that case right? The regenerated `config.yml` seems to have removed/changed more jobs than intended.

Reply:
Correct, we want to remove all linux conda jobs except `py3.8 cu116`. And the [config.yml for the current PR](https://github.com/pytorch/audio/blob/b2a6ee5a36afc6776de62daf3b105010a7b92b78/.circleci/config.yml#L1084-L1096) looks appropriate.